src/System.Runtime.Extensions/tests/System/Environment.SetEnvironmentVariable.cs

```diff
@@ -61,6 +61,7 @@ public void IncorrectVariableThrowsArgument()
         }
 
         [Theory]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot, "Environment Variables monidification isn't supported in Appx.")]
```

Comment:
That's even better, the point is that if I grep for `SkipOnTargetFramework.*uap` I see the reason right next to the 500 hits and don't have to open the file.